src/storage/src/types/sources.rs

```diff
@@ -868,7 +970,7 @@ impl UnplannedSourceEnvelope {
                     Some(desc) => desc,
                     None => {
                         return Ok((
-                            self.into_source_envelope(None, None),
+                            self.into_source_envelope(None, None, None),
```

Comment:
might want a comment here explaining that this case is only hit by the `None` case (or better yet, move them into different branches